app/views/manuals/_content_for_manual.html.erb

```diff
@@ -0,0 +1,7 @@
+<% content_for :extra_headers do %>
```

Comment:
I don't know about the name for this partial, content_for_manual makes it sound like it's writing out the content of the manual.  I get that content_for is a rails method and thats the only thing we do in this partial, but it doesn't really tell me what this partial is really for - should all content_for calls go in here?  That seems somewhat arbitrary.